rifle.py

```diff
@@ -36,17 +40,49 @@ def __init__(self, universe, DRAWING_SCALE):
         self.word_length_min_ = 3
         self.word_length_range = 3
 
-
+    def getTargetID(self,input): 
+        for enemy in self.universe_.enemies(): 
+            target_tag_word = self.target_tags_[enemy.ID_]
+            if(input == target_tag_word):
+                target_ID = self.ids_for_target_tags_[input]
+                break
+            else:       # input does not exist 
+                target_ID = -1
+        return target_ID  
+
+    def checkIfInputMatchesTargetTag(self): 
+        target_ID =  self.getTargetID(self.current_text_)
+        if (target_ID != -1): 
+            target_position = self.universe_.enemies_[target_ID].position_  
+            return target_position 
+        else:
+            return -1
     """
     Update Functions
     """
-
     def update(self, events):
         for enemy in self.universe_.enemies():
             if enemy.ID_ not in self.target_tags_:
-                self.target_tags_[enemy.ID_] = \
-                            self.word_generator_.request_word(self.word_length_min_, self.word_length_range)
-
+                new_word = self.word_generator_.request_word(self.word_length_min_, self.word_length_range)
+                self.target_tags_[enemy.ID_] = new_word
+                self.ids_for_target_tags_[new_word] = enemy.ID_ 
+
+        for event in events:
+            if event.type == pygame.KEYDOWN:
+                if event.key == EVENT_KEY_ENTER:
+                    target_location = self.checkIfInputMatchesTargetTag()
+                    self.current_text_ = ""
+                    if (target_location == -1):
+                        pass
```

Comment:
In general, if statements should be not be empty. This `if else` block could be better written as:
`if target_location != -1:`
`     your code here`